cmd/podman/common.go

```diff
@@ -454,9 +454,9 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"sysctl", []string{},
 		"Sysctl options (default [])",
 	)
-	createFlags.Bool(
-		"systemd", cliconfig.DefaultSystemD,
-		"Run container in systemd mode if the command executable is systemd or init",
+	createFlags.String(
+		"systemd", "true",
+		`Run container in systemd mode if the command executable is systemd or init ("true"|"false"|"always" (default "true")`,
```

Comment:
Probably should take out the 'systemd or init' part here, since `always` ignores it.